packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx

```diff
@@ -229,13 +231,13 @@ const PostsPagePostHeader = ({post, answers = [], toggleEmbeddedPlayer, hideMenu
               </NewFeaturePulse>
             )
           }
-          <div className={classes.commentsLink}>
+          {post.startTime && <div className={classes.commentsLink}>
             <AddToCalendarButton post={post} label="Add to Calendar" hideTooltip={true} />
-          </div>
+          </div>}
```

Comment:
This button's font size is too small, plus we prob want to adjust the spacing between the icon and the label to be consistent:
<img width="647" alt="Screen Shot 2023-02-13 at 4 08 37 PM" src="https://user-images.githubusercontent.com/9057804/218575593-0ff2c877-f145-465c-9a6c-0813f820d980.png">